launchable/commands/record/build.py

```diff
@@ -52,6 +52,9 @@
 @click.pass_context
 def build(ctx, build_name, source, max_days, no_submodules,
           no_commit_collection):
+    if "/" in build_name:
+        exit("Please pass build name without forward slash '/' to --name.")
```

Comment:
Since this check is not only about a forward slash, let's change this error message a bit, like `--name must not contain a slash`.

Reply:
updated https://github.com/launchableinc/cli/pull/292/commits/3b4de1a19711e8215db56301aad11ccdcb7ec0ac